bin/generate-stories.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+
+const dextroseExtension = '.dextrose.tmp.js';
+
+const overRideStories = (dirPath, files, stringToFind, stringToReplace) =>
+  new Promise((resolve, reject) => {
+    files.forEach((file) => {
+      fs.readFile(file, 'utf8', (err, data) => {
+        if (err) {
+          reject(err);
+        }
+        if (data.includes(stringToFind)) {
+          const updatedData = data.replace(stringToFind, stringToReplace);
+          const trimmedFile = file.replace('.js', '');
```

Comment:
can't you use this instead and get rid of the next line?

```const trimmedFile = file.replace('.js', dextroseExtension)```

Reply:
👍